packages/bpk-component-price-range/src/BpkPriceRange.module.scss

```diff
@@ -0,0 +1,134 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:math';
```

Comment:
This is going to cause issues for those not yet migrated to dart-sass as thisis not supported in node-sass, so for now we should change this to not use sass:math to use dividing / until the company is ready with dart-sass

It will throw warnings but this is unfortunately a pain point until node-sass is fully gone - I will check in the enablement team to see where we are at but for now I would avoid using this function :)